rpcs3/Emu/FS/VFS.cpp

```diff
@@ -379,7 +381,35 @@ vfsDevice* VFS::GetDevice(const std::string& ps3_path, std::string& path) const
 		}
 
 		if (max_i < 0)
-			return nullptr;
+		{
```

Comment:
@Nekotekina See Buzz!(TM) Junior: RoboJam (NPEA00093). Other games of Buzz!(TM) series,  also seem to exhibit this behaviour. I'll also write a test, as you requested.